### ISSUE TITLE
Instability of training results

### ISSUE THREAD

**Issue by zzdj11** [QIFEIDKN/STAGATE_pyG#2]
Hello, when I run with the pytorch version, I found that even after setting the random seed, I still can't make the result stable, I don't know what's the reason, I hope to get your help, thanks!

**Comment by forechoandlook** [comment]
> 你好，我用pytorch版本运行的时候，发现即使设置了随机种子，还是不能让结果稳定，不知道是什么原因，希望能得到你的帮助，谢谢！

Hello, what is your result with this repo? My reproducation result is `ARI 0.29` in `151676`.

**Comment by forechoandlook** [comment]
The result has been advanced by this repo `https://github.com/QIFEIDKN/STAGATE_pyG-Sigmoid`

**Comment by zzdj11** [comment]
> > 你好，我的pytorch版本运行的时候，发现即使设置了可以随时使用，还是不能让稳定，不知道是什么原因，希望得到你的帮助，谢谢！
> 
> 你好，你对这个 repo 的结果是什么？我的复制结果`ARI 0.29`在`151676`.



> > 你好，我用pytorch版本运行的时候，发现即使设置了随机种子，还是不能让结果稳定，不知道是什么原因，希望能得到你的帮助，谢谢！
> 
> Hello, what is your result with this repo? My reproducation result is `ARI 0.29` in `151676`.

Hello! My replication results are very unstable, the ARI results on the 151767 dataset are between 0.50-0.65 and I still can't make the final results stable by random seeding.

**Comment by zzdj11** [comment]
> > 你好，我用pytorch版本运行的时候，发现即使设置了随机种子，还是不能让结果稳定，不知道是什么原因，希望能得到你的帮助，谢谢！
> 
> Hello, what is your result with this repo? My reproducation result is `ARI 0.29` in `151676`.

![image](https://user-images.githubusercontent.com/69029746/179983708-93af9dbe-ea62-4986-b14c-417ec600ea30.png)

**Comment by zzdj11** [comment]
> > 你好，我用pytorch版本运行的时候，发现即使设置了随机种子，还是不能让结果稳定，不知道是什么原因，希望能得到你的帮助，谢谢！
> 
> Hello, what is your result with this repo? My reproducation result is `ARI 0.29` in `151676`.

Hello! My replication results are very unstable, the ARI results on the 151767 dataset are between 0.50-0.65 and I still can't make the final results stable by random seeding.

**Comment by forechoandlook** [comment]
> > > 你好，我用pytorch版本运行的时候，发现即使设置了随机种子，还是不能让结果稳定，不知道是什么原因，希望能得到你的帮助，谢谢！
> > 
> > 
> > Hello, what is your result with this repo? My reproducation result is `ARI 0.29` in `151676`.
> 
> Hello! My replication results are very unstable, the ARI results on the 151767 dataset are between 0.50-0.65 and I still can't make the final results stable by random seeding.

Can you show your code by google drive?

**Comment by zhaofangyuan98** [comment]
Hello, could you tell me your env version(python, cuda, pytorch, torch-geometric, torch-scatter,  torch-sparse and so on)? Thank you very much!
